pkg/controller/certapi/certapi.go

```diff
@@ -78,19 +78,21 @@ func New(ctx context.Context, config *config.APIServerConfig, db *database.Datab
 }
 
 // Parses and validates the token against the configured keyID and public key.
-// If the token si valid the token id (`tid') and subject (`sub`) claims are returned.
-func (c *Controller) validateToken(ctx context.Context, verToken string, publicKey crypto.PublicKey) (string, *database.Subject, error) {
+// A map of valid 'kid' values is supported.
+// If the token is valid the token id (`tid') and subject (`sub`) claims are returned.
+func (c *Controller) validateToken(ctx context.Context, verToken string, publicKeys map[string]crypto.PublicKey) (string, *database.Subject, error) {
```

Comment:
This is probably fine for now, but this looks awfully like the publicKeyCache

Reply:
it's populated from the public key cache - see certificate.go, line 63